### PR TITLE
chore(guides): update Web Modeler CI/CD guide for 8.2

### PR DESCRIPTION
**Pull request by wollefitz**
## Description

Follow-up for https://github.com/camunda/camunda-docs/pull/2770, see https://github.com/camunda/camunda-docs/pull/2770#issuecomment-1777891039.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md [hunk 1]
```diff
@@ -82,90 +82,17 @@ Synchronize files between Web Modeler and version control systems (VCS) and vice
...
-  "size": 50
-}
-```
-
-:::info
-Pagination is enforced for all listed `search` endpoints. Ensure you obtain all relevant pages.
-:::
-
-We work to replace this with a webhook or subscription approach. An alternate approach involves manually triggering synchronization or delegating synchronization triggers to other sources, such as the pipeline itself, creation of new branches, or pull/merge requests.
+To listen to changes, starting from 8.3 Web Modeler offers a polling approach that compares the update dates with the last sync dates recorded, see [the 8.3 documentation](/versioned_docs/version-8.3/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md#sync-files-with-version-control).
```
Comment (akeller) [inline versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md:85]:
```suggestion
Starting in 8.3, Web Modeler offers a polling approach that compares the update dates with the last sync dates recorded, see [the 8.3 documentation](/versioned_docs/version-8.3/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md#sync-files-with-version-control).
```

Comment (akeller) [reply versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md:85]: @pepopowitz is this going to cause problems in the future? If we want to link across versions, should we? Or should we ask users to use the version picker in the navbar?

Comment (pepopowitz) [reply versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md:85]: I think this is the best way to approach this. 

Asking the user with text in the docs to pick a version from the navbar feels like putting a burden on them that I don't think is necessary, and it relies on them to interpret the instructions correctly.

Linkng to a specific version makes sense, based on the text, and at archival time it shouldn't be a problem: 

* When this version (8.2) is archived, this link will show up as broken. We'll have to fix it, by making it a fully qualified URL to `https://docs.camunda.io/8.3/...`. This seems like a thing we should have to do (preserve links), and it's very minor.
* Later, when 8.3 archives, anything at `docs.camunda.io/8.3/` will be redirected to `unsupported.docs.camunda.io/8.3/`, and the link from the 8.2 site will Just Work ™️ . 

I don't think there are a lot of times where we will _want_ to link across versions; but I think this is a good example of a reason to do it, and it's the correct approach.

**Review by akeller — APPROVED** [review]

File: versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md (hunk 1 above)
Comment (akeller) [reply versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md:85]: Ignore my suggestion here, approving and merging.

**Pull request merged**